apps/asap-server/src/entities/user.ts

```diff
@@ -75,6 +75,15 @@ export const userUpdateSchema = Joi.object({
   skills: Joi.array().items(Joi.string()),
   skillsDescription: Joi.string().allow(''),
   questions: Joi.array().items(Joi.object({ question: Joi.string() })),
+  teams: Joi.array().items(
+    Joi.object({
+      id: Joi.string().required(),
+      responsibilities: Joi.string(),
+      approach: Joi.string(),
+    })
+      .min(2)
+      .required(),
```

Comment:
Will this mean we always have to provide teams in a patch request?

Reply:
No, just that the teams array has to have an object in it